syntax = "proto3";

import "graph.proto";

package proto_router;

message Settings {
    uint32 bus_wait_time = 1;
    double bus_velocity = 2;
}

message TransportRouter {
    Settings settings = 1;
    proto_graph.Router router = 2;
}
